### PR TITLE
Likes: Correctly size user profile sheet pop over on iPad.

### LINKED ISSUE #16244: Notification Likes List: show user profile when user tapped
In Notifications > Likes > List, when a user row is tapped, show the user profile in a bottom sheet.

References:

- Likes Enhancements i2 design - pbArwn-1CQ-p2
- User Profiles i3 design - pbArwn-K3-p2

user_profile

### PR DESCRIPTION
**Pull request by aerych**
Closes #16244 

This PR applies changes to correctly size the likes user profile sheet when shown in an popover on the iPad. 

To test:
View the patch on an iPad (real or simulator). 
View different user profiles and confirm the popover is correctly sized to match.
View other bottom sheet popovers shown in the reader and when publishing and confirm there is no regression in sizing those popovers.

## Regression Notes
1. Potential unintended areas of impact
Bottom sheets presented by the Reader menu (e.g. following and p2 list) and when publishing posts.

2. What I did to test those areas of impact (or what existing automated tests I relied on)
Tested bottom sheets in the reader and when publishing in the simulators (iPhone and iPad) and confirmed they were sized correctly. 

3. What automated tests I added (or what prevented me from doing so)
None. 

PR submission checklist:

- [x] I have completed the Regression Notes.
- [x] I have considered adding unit tests for my changes.
- [x] I have considered adding accessibility improvements for my changes.
- [x] I have considered if this change warrants user-facing release notes and have added them to `RELEASE-NOTES.txt` if necessary.

### CODE REVIEW

File: WordPress/Classes/ViewRelated/User Profile Sheet/UserProfileSheetViewController.swift [hunk 1]
```diff
@@ -34,19 +34,10 @@ class UserProfileSheetViewController: UITableViewController {
...
         presentedVC?.presentedView?.layoutIfNeeded()
     }
 
-    // We are using intrinsicHeight as the view's collapsedHeight which is calculated from the preferredContentSize.
-    override var preferredContentSize: CGSize {
-        set {
-            // no-op, but is needed to override the property.
-        }
-        get {
-            return UIDevice.isPad() ? Constants.iPadPreferredContentSize : tableView.contentSize
```
Comment (ScoutHarris) [inline WordPress/Classes/ViewRelated/User Profile Sheet/UserProfileSheetViewController.swift:46]: Constants.iPadPreferredContentSize can be removed as well.

**Review by ScoutHarris — APPROVED** [review]
Tested on iPad and iPhone. Looks great! Thank you so much for fixing this!!!

I left one nitpick, but otherwise :shipit: .

File: WordPress/Classes/ViewRelated/User Profile Sheet/UserProfileSheetViewController.swift (hunk 1 above)
Comment (aerych) [reply WordPress/Classes/ViewRelated/User Profile Sheet/UserProfileSheetViewController.swift:46]: Ah yes!  I shall nuke it! Good catch

**Comment by aerych** [comment]
Thanks @ScoutHarris !!

**Pull request merged**